.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,14 +7,10 @@ If something doesn't apply please check the box and add a justification after th
 -->
 
 - [ ] Link to docs PR or issue --
-- [ ] Link to UI issue or PR --
-- [ ] Is the [issue worked on linked][1]? --
-- [ ] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) --
-- [ ] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
-- [ ] Unit Tests --
-- [ ] E2E Tests --
-- [ ] Manual Universal Tests --
-- [ ] Manual Kubernetes Tests --
+- [ ] Link to UI issue --
```

Comment:
I'd remove this. 99% PRs are not relevant to the GUI.

Reply:
Did some collapsing as suggested by @michaelbeaumont